microcosm_pubsub/dispatcher.py

```diff
@@ -49,7 +49,11 @@ def handle_message(self, message, bound_handlers) -> MessageHandlingResult:
                     message=message,
                     bound_handlers=bound_handlers,
                 )
+            # save elapsed time
             message_handling_result.elapsed_time = self.opaque["elapsed_time"]
+            # log handler outcome
```

Comment:
We now log after creating the message result.

There is still some awkwardness with this logic. It'll need one more pass I think.